vision/draw_box_around_faces.rb

```diff
@@ -0,0 +1,79 @@
+# Copyright 2016 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START all]
+# [START import_client_library]
+require "google/cloud/vision"
+# [END import_client_library]
+# [START import_rmagick]
+require "rmagick"
+# [END import_rmagick]
+
+# [START detect_faces]
+def draw_box_around_faces path_to_image_file:, path_to_output_file:
+  # [START get_vision_service]
+  vision = Google::Cloud::Vision.new
```

Comment:
FYI This PR made no changes to this existing tutorial.

I'll update this tho for consistency